src/Async.js

```diff
@@ -145,10 +145,10 @@ export default class Async extends Component {
 		const props = {
 			noResultsText: isLoading ? loadingPlaceholder : searchPromptText,
 			placeholder: isLoading ? loadingPlaceholder : placeholder,
-			options: isLoading ? [] : options,
+			options: (isLoading && loadingPlaceholder) ? [] : options,
```

Comment:
If your autocomplete service is fast, A) you don't need a loading placeholder, and B) setting options to [] causes all the items to 'flicker' out before reappearing.